JitsiConference.js

```diff
@@ -968,24 +1147,9 @@ JitsiConference.prototype.onCallEnded
         this.statistics.stopCallStats();
     }
     // Current JingleSession is invalid so set it to null on the room
-    this.room.setJingleSession(null);
+    this.jingleSession = null;
```

Comment:
For me it fells weird that ChatRoom was involved in dealing with the JingleSession. I guess it was because we were missing some information for creating remote tracks like which user owns which track, video type etc. I created a SignallingLayer to be able to split that out. If you look at code in the master branch you will see that it's not doing anything useful with the session, but being just another "hop" in the execution chain.